tensorflow/core/grappler/optimizers/meta_optimizer.cc

```diff
@@ -149,16 +149,14 @@ bool IsXlaGlobalJitOn(
 
 // A helper function to decide whether to enable the memory optimizer.
 bool MemoryOptimizerEnabled(
-    RewriterConfig::MemOptType mem_opt_type,
-    OptimizerOptions::GlobalJitLevel jit_level_in_session_opts) {
+    RewriterConfig::MemOptType mem_opt_type, bool xla_on) {
```

Comment:
I would call this parameter `xla_auto_clustering_on` since we're not checking for other ways to enable XLA.

Reply:
done